source/gloperate/source/painter/AbstractOutputCapability.cpp

```diff
@@ -0,0 +1,33 @@
+#include <gloperate/painter/AbstractOutputCapability.h>
+
+#include <gloperate/base/collection.hpp>
+
+#include <gloperate/pipeline/AbstractData.h>
+#include <gloperate/pipeline/AbstractStage.h>
+
+namespace gloperate
+{
+
+/**
+*  @brief
+*    Constructor
+*/
+AbstractOutputCapability::AbstractOutputCapability()
+    : AbstractCapability()
```

Comment:
@cgcostume now prefers uniform initialization where applicable, so here it would be
```c++
    : AbstractCapability{}
```